infra/modules/notifications/data/main.tf

```diff
@@ -0,0 +1,11 @@
+locals {
+  stripped_domain_name = replace(var.domain_name, "/[.]$/", "")
```

Comment:
I see this stripping of the domain name a number of times in this PR, but don't understand why it's necessary. service_config.domain_name doesn't have a trailing period.

Reply:
Huh! Not sure why my source examples had it then. Good catch